distribution/docs/src/main/resources/content/_configuring/reusing-configurations.adoc

```diff
@@ -7,7 +7,7 @@
 
 == {title}
 
-The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state for either a brand new installation or a second node for a highly available deployment.
+The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state for either a brand new installation or a second node for a Highly Available Cluster.
 
 To export the current configuration settings:
 
```

Comment:
🖊 Use back-ticks around `migration:export` on the next line.

Reply:
For the record, this was not a part of this PR! It's an easy enough fix though, so I'll allow it.

Reply:
@lessarderic For the record, I was asked to remove the back-ticks on my PR

Reply:
Works for me either way, as long we're consistent.